packaging/installer/kickstart.sh

```diff
@@ -877,6 +878,24 @@ try_package_install() {
       uninstall_subcmd="remove"
       INSTALL_TYPE="binpkg-rpm"
       ;;
+    rocky)
+      if command -v dnf > /dev/null; then
+        pm_cmd="dnf"
+        repo_subcmd="makecache"
+      else
+        pm_cmd="yum"
+      fi
+      SYSVERSION_ROCKY="8"
+      #repo_prefix="el/${SYSVERSION}"
+      repo_prefix="el/${SYSVERSION_ROCKY}"
```

Comment:
- do we need the commented-out line?
- do we need `SYSVERSION_ROCKY` as a variable? My understanding is that is manually set and always 8.